rbi/annotations/devise.rbi

```diff
@@ -145,3 +145,26 @@ class Devise::UnlocksController < DeviseController
   sig { params(resource: T.untyped).returns(String) }
   def after_unlock_path_for(resource); end
 end
+
+# @shim: Devise controllers are loaded by rails
+class Devise::OmniauthCallbacksController < DeviseController
+  # GET|POST /resource/auth/provider
+  sig { returns(T.untyped) }
+  def passthru; end
+
+  # GET|POST /resource/auth/provider/callback
+  sig { returns(T.untyped) }
+  def failure; end
+
```

Comment:
I think we're missing `protected` here https://github.com/heartcombo/devise/blob/v4.9.4/app/controllers/devise/omniauth_callbacks_controller.rb#L15